pkg/platform/api/graphql/model/projectconv.go

```diff
@@ -41,6 +41,10 @@ func (p *Project) ToMonoProject() (*mono_models.Project, *failures.Failure) {
 		}
 	}
 
+	if p.RepoURL == nil {
+		return nil, failures.FailMarshal.Wrap(ErrMissingRepoURL)
+	}
```

Comment:
I tried to update this method to return an error but it quickly snowballed so opted to add a failure here instead.